docs/it/userinfo_endpoint.rst

```diff
@@ -22,9 +22,9 @@ Lo UserInfo Endpoint deve supportare l’uso del solo metodo HTTP GET :rfc:`2616
 Response
 ++++++++
 
-La response dello UserInfo Endpoint deve specificare nel “Content-Type” il valore “application/jwt”.
+La response dello UserInfo Endpoint deve specificare nel "Content-Type" il valore "application/jwt".
 
-Il contenuto trasmesso nel body della Response deve essere un JWT firmato e cifrato secondo le modalità definite dall’Agenzia per l’Italia Digitale.
+Il contenuto del corpo della Response deve essere un JWT firmato e cifrato secondo le `modalità definite dall'Agenzia per l'Italia Digitale. <https://openid.net/specs/openid-connect-core-1_0.html#UserInfoResponse>`_
```

Comment:
rimuovere "secondo le `modalità definite dall'Agenzia per l'Italia Digitale. " perché out of scope per CIE